content/en/docs/ibm/create-cluster.md

```diff
@@ -0,0 +1,138 @@
++++
+title = "Create an IBM Cloud cluster"
```

Comment:
Would this have been better handled by point to offical docs from IKS? Else we will be updating often?

Reply:
Yes and no, the official docs definitely cover all the scenarios including creating clusters through UI. But usual times it is difficult to follow what exact command and options should be used. A up-to-date tutorial would be better than the current official docs. I meant something like https://cloud.ibm.com/docs/containers?topic=containers-cs_cluster_tutorial is better than https://cloud.ibm.com/docs/containers?topic=containers-getting-started.

The commands in this docs are easier to follow.

But yes, we can look for ways to improve.